.gitignore

```diff
@@ -5,4 +5,4 @@ venv
 .DS_Store
 *.pyc
 test/sagemaker_tests/huggingface/inference/resources/**/*[.json,.h5,.bin,.txt]
-build/
+build/
```

Comment:
Could you remove this change so that there is no diff for the `.gitignore` file in this PR?